testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -110,7 +110,7 @@
 end
 
 When(/^I wait until I see the name of "([^"]*)", refreshing the page$/) do |host|
-  raise 'Overview System page didn\'t load' unless has_content?('Download CSV')
+  raise 'Overview System page didn\'t load' unless has_content?('Download CSV') || has_content?('System Overview') || has_content?('Systems') || has_content?('Keys')
```

Comment:
`has_content?('Systems')` is always visible in the left menu so this would always return true

Reply:
I have seen this comment from Maxime these days, too IIRC.

Reply:
Isn't that a bit too permissive ? What about for example reducing it to "System Overview" ?

Reply:
I need the "Keys", and this only searches inside of the spacewalk-content div, for reference

Reply:
so it doesn't see anything on the left side, in the menu

Reply:
It's odd, Maxime was saying that it was always visible for him https://github.com/uyuni-project/uyuni/pull/7419